src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -36,6 +36,31 @@ public AnalyzeCommandBaseTests(ITestOutputHelper output)
             Output.WriteLine($"The seed that will be used is: {TestRule.s_seed}");
         }
 
+        [Fact]
+        public void AnalyzeCommandBase_PerTargetAnalyzeEventsAreReceived()
+        {
+            int filesCount = Directory.GetFiles(Environment.CurrentDirectory, "*" ).Length;
+
+            var options = new TestAnalyzeOptions()
+            {
+                TargetFileSpecifiers = new List<string>(new[] { "*" })
+            };
+
+            var logger = new TestMessageLogger();
+
+            var context = new TestAnalysisContext
+            { 
+                Logger = logger,
+                MaxFileSizeInKilobytes = long.MaxValue,
```

Comment:
>MaxValue [](http://example.com/codeflow?start=46&length=8)

Set the max value so that we don't skip any files on enumerating the current working directory.